src/Esfa.Vacancy.UnitTests/CreateApprenticeship/Api/Orchestrators/GivenACreateApprenticeshipOrchestrator/WhenCreatingAnApprenticeshipVacancy.cs

```diff
@@ -33,6 +34,8 @@ public class WhenCreatingAnApprenticeshipVacancy
         private CreateApprenticeshipOrchestrator _orchestrator;
         private Mock<IValidationExceptionBuilder> _mockValidationExceptionBuilder;
         private string _expectedErrorMessage;
+        private Dictionary<string, string> _validHeader
+            = new Dictionary<string, string> { { Constants.RequestHeaderNames.ProviderUkprn, "12345678" } };
```

Comment:
the value, `12345678`, for this can be generated randomly from fixture.

Reply:
Converted into constant